changelog.d/8802.feature

```diff
@@ -0,0 +1 @@
+The prometheus rules are updated to include missing rules the grafana dashboard uses.
```

Comment:
I'd probably make this a `.misc` (translating to Internal Changes in the changelog) rather than a feature.

Reply:
Actually @auscompgeek is probably right. Since this isn't code and is just meant to be a copy-pastable file `.doc` is indeed best.